api/src/main/java/com/github/retrooper/packetevents/protocol/color/DyeColor.java

```diff
@@ -0,0 +1,103 @@
+package com.github.retrooper.packetevents.protocol.color;
+
+import net.kyori.adventure.util.RGBLike;
+import org.jetbrains.annotations.NotNull;
+
+
+/*
+ * This file is part of packetevents - https://github.com/retrooper/packetevents
+ * Copyright (C) 2024 retrooper and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+public enum DyeColor implements RGBLike {
+    WHITE(new Color(0xf9fffe), new Color(0xffffff), new Color(0xf0f0f0), 8),
```

Comment:
And why is there always an empty line between each enum type, that doesn't necessarily make it more readable for me, rather more filled and therefore more confusing

Reply:
I fixed it just for you pookiebear

Reply:
Either my eyes are bad or it's still the same